app/src/features/Batches/BatchList.tsx

```diff
@@ -58,6 +59,10 @@ const useStyles = makeStyles((theme) => ({
 
 const PAGE_SIZE = 10;
 
+const createKibanaLink = (batchCreation: string) => {
+  return `${KIBANA_URL}app/kibana#/discover?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-1y,to:now))&_a=(columns:!(_source),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:ffb9f770-148b-11ec-afc5-23ae59245f97,key:'@timestamp',negate:!f,params:(query:'${batchCreation}'),type:phrase),query:(match_phrase:('@timestamp':'${batchCreation}')))),index:ffb9f770-148b-11ec-afc5-23ae59245f97,interval:auto,query:(language:kuery,query:''),sort:!())`;
```

Comment:
this doesn't seem right:
- `time:(from:now-1y,to:now)` should be something like `time:(from:${batchCreation},to:${batchEnd || 'now'})`
- `index:ffb9f770-148b-11ec-afc5-23ae59245f97` must be removed
- `params:(query:'${batchCreation}'),type:phrase),query:(match_phrase:('@timestamp':'${batchCreation}'))))` --> this is not how you search for a date (cf point number 1). However you should search for the string "error" in the logs